File: dsa_js/array_max_element.js
function findMaxElement(arr) {
    let maxElement = arr[0];
    for(let i = 0; i < arr.length; i++) {
        if (arr[i] > maxElement) {
            maxElement = arr[i];
        }
    }
    return maxElement;
}
const list = [7, 12, 9, 4, 11]
console.log(findMaxElement(list));

File: dsa_python/array_max_element.py
def find_max_element(arr):
    max_element = arr[0]
    for i in range(1, len(arr)):
        if arr[i] > max_element:
            max_element = arr[i]
    return max_element

num_list = [7, 12, 9, 4, 11]
print(find_max_element(num_list))